### PR TITLE
Fixes #33647 - Return persisted attrs on edit mutation

### PR DESCRIPTION
**Pull request by xprazak2**

### CODE REVIEW

**Comment by xprazak2** [comment]
cc @timogoebel @kamils-iRonin for visibility

File: app/graphql/mutations/base_mutation.rb [hunk 1]
```diff
@@ -62,6 +62,7 @@ def save_object(resource)
         errors = if resource.save
                    []
                  else
+                   resource.reload if resource.persisted?
```
Comment (kamils-iRonin) [inline app/graphql/mutations/base_mutation.rb]: is hit the db necessary? maybe `restore_attributes` would be enough?
```suggestion
                   resource.restore_attributes if resource.persisted?
```

Comment (xprazak2) [reply app/graphql/mutations/base_mutation.rb]: `restore_attributes` would be enough, thanks.

**Review by kamils-iRonin — APPROVED** [review]

**Review by timogoebel — APPROVED** [review]

**Comment by timogoebel** [comment]
[test katello]

**Comment by timogoebel** [comment]
[test katello]

**Comment by timogoebel** [comment]
Thanks!

**Comment by ezr-ondrej** [comment]
Do we want to cherry-pick @xprazak2 ?

**Comment by xprazak2** [comment]
I do not think we need to, it does not affect an existing feature as far as I know.

**Pull request merged**